toolchain/check/declaration_name_stack.h

```diff
@@ -11,6 +11,19 @@
 
 namespace Carbon::Check {
 
+// An index for a pushed scope, which may be a permanent scope with a
+// corresponding `NameScope`, or a temporary scope, such as is created for a
+// block.
+//
+// These are comparable, and lower values correspond to scopes entered earlier
+// in the file.
```

Comment:
Does "These" mean "indices" or "scopes"? Does "lower values" refer to values in lowering, the indexed values, or the comparison of indices?

Maybe:

```suggestion
// Indices are comparable. Lower indices correspond to scopes entered earlier in the file.
```

Reply:
Fixed, and avoided taking a position on "indexes" versus "indices" ;)

Reply:
Lies! "`ScopeIndex` is comparable" would avoid taking a stance. :-P

Reply:
Sure, changed to "`ScopeIndex` values are" to further sidestep the question.

(For what it's worth, I do think that "`ScopeIndex`es" is not taking a position on "indexes" vs "indices" here. I would write -- and in other codebases, have written -- "`Child`s are ..." too, and I would use "`ScopeIndex`es" here even if we decide we want to use "indices" in general. I don't think it makes sense to apply irregular plurals to quoted technical text -- "`ScopeIndices`" or some attempt at a middle ground like "`ScopeIndic`es" would obviously be wrong.)